src/server/server.py

```diff
@@ -121,46 +59,74 @@ def create(
         expiration: Optional[float] = None,
         prefetch_batches: int = 1,
         sender_threads: int = 1,
-        max_block_selection_delay: float = 1,
+        mean_block_selection_delay: float = 0.5,
+        mean_balance_check_period: float = 150,
+        min_balance_quality: float = 0.8,
```

Comment:
```suggestion
        min_balance_quality: float = 0.0,
```

TODO: Disable rebalancing by default, unless we solve issues with shmem